epp-client/src/connection/client.rs

```diff
@@ -46,7 +46,6 @@
 //! }
 //! ```
 
-use futures::executor::block_on;
```

Comment:
I'm on my phone at the moment so I might be wrong but can we remove the `futures` dependency from Cargo.toml?

Reply:
Yeah the dependency can be removed.